debian/dist/Ubuntu/control

```diff
@@ -162,7 +163,8 @@ Description: MariaDB database client binaries
 
 Package: mariadb-server-core-10.1
 Architecture: any
-Depends: ${shlibs:Depends}, ${misc:Depends}, libmariadbclient18 (>= ${binary:Version})
+Depends: ${shlibs:Depends}, ${misc:Depends}, libmariadbclient18 (>= ${binary:Version}),
+    libsystemd-daemon0
```

Comment:
So you reverted this. Please note it was rather a question. I'm not sure that dependency is added automatically (though I believe it should). I assume you tested it.